src/routes/manage-selections/components/SelectionItem.tsx

```diff
@@ -62,7 +61,7 @@ const SelectionItem = ({ identifier, deleteItem }: PropsType): JSX.Element => {
         <Explanation>{labels.manageSelection.deleteModal.explanation}</Explanation>
       </Modal>
       <ListItem
-        title={data.title}
+        title={data?.title ?? labels.general.error.unknown}
```

Comment:
I think it would really make sense to also show this ListItem with the delete-icon on `Forbidden`-error